modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -119,6 +107,57 @@ If you modified a document stored in Infinite Scale, you can always restore olde
 
 // FIXME: Elaborate more when it's working. Is there a diff to come? I hope so.
 
+== Sharing Roles and Permissions
+
+* Members can be added to a Space so that they gain access to all files and folders within the space. Members get the sharing *and* space permissions granted depending by their role.
```

Comment:
```suggestion
* Members can be added to a space so that they gain access to all files and folders within the space. Members are granted sharing *and* space permissions, depending on their role.
```